modules/Bio/EnsEMBL/Taxonomy/PipeConfig/ImportNCBItaxonomy_conf.pm

```diff
@@ -48,219 +30,216 @@ package Bio::EnsEMBL::Taxonomy::PipeConfig::ImportNCBItaxonomy_conf;
 use strict;
 use warnings;
 
-use Bio::EnsEMBL::Hive::Version 2.3;
-
 use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
 
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
```

Comment:
For my personal knowledge, why aren't we using standard ProductionHive parent here?

Reply:
It's in a different repo, so it perhaps is a bit less obfuscated about what parameters are set; and this pipeline is very unlike other Production pipelines, in terms of its structure, so inheriting from the Production parent doesn't really provide much benefit.

Reply:
Ok no problem. Thanks for the details.